wrappers/ld-wrapper.sh

```diff
@@ -26,10 +26,17 @@ if [ "$TARGET" = "$BASENAME" ]; then
     TARGET=$DEFAULT_TARGET
 fi
 ARCH="${TARGET%%-*}"
+TARGET_OS="${TARGET##*-}"
 case $ARCH in
 i686)    M=i386pe   ;;
 x86_64)  M=i386pep  ;;
 armv7)   M=thumb2pe ;;
 aarch64) M=arm64pe  ;;
 esac
-ld.lld -m $M "$@"
+FLAGS="-m $M"
+case $TARGET_OS in
+mingw32uwp|mingw32winrt)
+	FLAGS="$FLAGS -lmincore"
```

Comment:
This seems to use a different indentation level than the rest. (I've tried to use 4 spaces for indentation in the scripts and example code.)

Reply:
Fixed